src/sharedInterfaces/connectionGroup.ts

```diff
@@ -41,6 +41,7 @@ export interface ConnectionGroupState {
     description?: string;
     color?: string;
     message?: string;
+    parentId?: string;

```

Comment:
Why does this need to be added for this showing the OE options?  The state should only need the info necessary for the UI, and the connection dialog UI doesn't (currently) have an option for setting the parent when creating a new connection group.